### PR TITLE
Add python module "requests"

### PR DESCRIPTION
**Pull request by jngrad**
Fix silent regressions from #144:
- `cuda:9` needs `requests` for the `style_doxygen` job
- `cuda:10.1` needs to set the `$PATH` for sphinx to work (now tested)

### CODE REVIEW

**Comment by jngrad** [comment]
@RudolfWeeber This is needed to make CI work again on the espresso repo

File: docker/ubuntu-python3/Dockerfile-cuda-9.0 [hunk 1]
```diff
@@ -22,6 +23,7 @@ RUN apt-get update && apt-get install -y \
     vim \
     ccache \
     graphviz \
+&& pip3 install requests \
```
Comment (RudolfWeeber) [inline docker/ubuntu-python3/Dockerfile-cuda-9.0]: Isn't a --user missing?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -163,6 +163,12 @@ test/ubuntu-python3:cuda-9.0:
 test/ubuntu-python3:cuda-10.1:
   extends: .test_template
   needs: ["ubuntu-python3:cuda-10.1:build"]
+  script:
```
Comment (RudolfWeeber) [inline .gitlab-ci.yml]: Why did you introduce these changes? Is that related to requests?

Comment (jngrad) [reply .gitlab-ci.yml]: here I'm testing that sphinx can actually run; we merged the previous PR without realizing the sphinx executables installed in the user directory were not accessible in the PATH, causing the CI failures in the espresso repo when building the sphinx docs

File: docker/ubuntu-python3/Dockerfile-cuda-9.0 (hunk 1 above)
Comment (jngrad) [reply docker/ubuntu-python3/Dockerfile-cuda-9.0]: This one is installed as root; didn't bother checking if it can be installed as user with some environment variable magic. When building docker images on my machine, pip has no issue installing packages as user, but on elk with the same Dockerfile, pip may fail if I don't manually set up environment variables. See for example 4057c0e29 that solved [this build failure](https://gitlab.icp.uni-stuttgart.de/espressomd/docker/-/jobs/181804) on Ubuntu 16.04, while the cuda 10.1 image on Ubuntu 18.04 doesn't have this issue. I'm not sure what's the real cause, though.
